### DISCUSSION TITLE
Modify registry uninstall information

### DISCUSSION THREAD

**Discussion by ad-chuliang** [wixtoolset/issues#8656] [Questions]
How to set UninstallString value at the path of \HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\[ProductCode]

**Comment by robmen** [comment]
Why do you want to do that?

**Reply by ad-chuliang** [reply]
I want to run the dialog defined in the msi when uninstalling a programme from the control panel, rather than just uninstalling it without any interaction. I checked and learnt that the command to run it is in the registry. Root is HKLM, Key is \SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall or SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall. After installing with msi, it is true that wix automatically writes the uninstall information under these paths, but after I manually change the value of UninstallString to MsiExec.exe /i[ProductCode], uninstalling in the control panel still doesn't pop up the msi Maintenance screen. And I tried to change the value of UninstallString in wix using RegistryKey and other elements, but the registry still defaults to MsiExec.exe /X[ProductCode] , not MsiExec.exe /i[[ProductCode], which is what I set in RegistryKey.
